### PR TITLE
fix: prevent to occur scrollEnd event when scrolling horizontally

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* related issue(#982)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/cypress/integration/eventBus.spec.ts [hunk 1]
```diff
@@ -183,20 +184,39 @@ it('columnResize', () => {
...
+
+  it('should not occur scrollEnd event after scrolling horizontally ', () => {
+    const callback = cy.stub();
+
+    cy.gridInstance().invoke('on', 'scrollEnd', callback);
+
+    // scroll at the bottommost
+    cy.focusToBottomCell(19, 'name');
+    // scroll horizontally
+    cy.focusToBottomCell(19, 'age');
```
Comment (jung-han) [inline packages/toast-ui.grid/cypress/integration/eventBus.spec.ts:216]: 이름을 `focusToBottomCell` 대신 `focus`로 해서 주석을 다는건 어떤가요?
`focusToBottomCell`이지만 실제로 이 커맨드가 다르게 쓰일 수도 있을 것 같아서요

**Comment by jung-han** [comment]
Run tests

**Comment by jung-han** [comment]
ok to test

**Comment by seonim-ryu** [comment]
ok to test

**Comment by seonim-ryu** [comment]
ok to test

**Comment by jung-han** [comment]
ok to test

File: packages/toast-ui.grid/cypress/integration/eventBus.spec.ts (hunk 1 above)
Comment (js87zz) [reply packages/toast-ui.grid/cypress/integration/eventBus.spec.ts:216]: 실제 렌더링 되는 시점까지 기다리는 로직이 있어 `focusAndWait`으로 변경하여 사용하겠습니다.

**Pull request merged**